src/redux/configureStore.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers, createStore } from 'redux';
+import booksReducer from './books/books';
+import categoriesReducer from './categories/categories';
+
+const rootReducer = combineReducers({ booksReducer, categoriesReducer });
```

Comment:
- [ ] Please give a key to your reducers for example
- books: bookReducer
- status: statusReducer

Reply:
Thank you so much for your feedback @GzimAsani! I will implement this change as you have asked.  😎 👍